core/server/server.spec.js

```diff
@@ -23,6 +23,14 @@ describe('The server', function() {
     server = undefined
   })
 
+  it('should allow strings for port', async function() {
+    //fixes #4391 - This allows the app to be run using iisnode, which uses a named pipe for the port.
+    const pipeServer = createTestServer({
+      port: '\\\\.\\pipe\\9c137306-7c4d-461e-b7cf-5213a3939ad6',
+    })
+    expect(pipeServer).to.not.undefined
```

Comment:
Not sure if this is a Chai shortcut that I'm just not familiar with or if it should have a `.be` in the chain?

```suggestion
    expect(pipeServer).to.not.be.undefined
```